### ISSUE TITLE
[🕹️] Create logo for Papermark 

### ISSUE THREAD

**Issue by shnai0** [mfts/papermark#630]
Papermark don't have a logo! So we would love to have one. 

1. Submit a logo design -> Get 150 points

2. You logo will be part of public vote on Twitter, so submit your Twitter name if you want to be mentioned 

3. Winner get additional -> 750 points

**Comment by Ionfinisher** [comment]
Hello @shnai0 , is there any particular directions for creating the logo? I would like to work on it. Thanks.

**Comment by mfts** [comment]
@Ionfinisher 

The logo should be simple and relate to Papermark or the Papermark product. 

Currently we only have a Wordmark of the text Papermark. 

Valid logos would be 
- square icon 
- designed wordmark of Papermark 
- icon + wordmark

Please align with the color scheme of Papermark, which is primarily black and white but has an orange accent color

**Comment by Ionfinisher** [comment]
Hi @mfts and @shnai0 , I actually have created two differents logos. Should I submit each logo in different PR?

**Comment by adarsh79singh** [comment]
Hi @mfts , 
I've just designed a new logo, and I'd love your thoughts on it. Does it align with the brand/image, and is there anything you think I should improve or change?

![paper Mark logo white](https://github.com/user-attachments/assets/ac8026c6-f218-467b-991b-6ce9f13f10fd)
![paper Mark logo black](https://github.com/user-attachments/assets/744447c1-8fba-4115-90e0-a87dcd41ea37)

**Comment by mfts** [comment]
@adarsh79singh could you please open a new issue and assign it to yourself. I can then award you there.

**Comment by adarsh79singh** [comment]
@mfts I’ve opened the issue and assigned it to myself as requested. You can now award me there whenever you're ready. Let me know if you need any further details!

> @adarsh79singh could you please open a new issue and assign it to yourself. I can then award you there.

**Comment by harshsbhat** [comment]
@mfts  Multiple PRs are allowed for most of the side quests right? I am guessing multiple logo submissions are allowed too

**Comment by Ionfinisher** [comment]
@mfts I think there should be a clarification on how we can submit this side quest because it's a little bit messy for me.

**Comment by FurquanAnwer** [comment]
Yes someone please explain how is someone supposed to solve these side
quests?

**Comment by mfts** [comment]
@Ionfinisher @FurquanAnwer you can open a new issue on github with the logo and then assign yourself. 

I will award you there.

**Comment by Ionfinisher** [comment]
Hello @mfts here is my issue: #662.

**Comment by harshsbhat** [comment]
@mfts Don't you think creating a PR with a line in oss.gg folder is a better option than creating an issue? Creating and assigning ourselves to that issue pretty much blocks us from solving/assigning ourselves in some code based issue.

**Comment by mfts** [comment]
@harshsbhat Oh right. I forgot about the blocking when assigned. Then PR also works.

**Comment by EzyHow** [comment]
Submitted new logo [here](https://github.com/mfts/papermark/issues/960). Please check.
